lib/cell.rb

```diff
@@ -28,8 +28,8 @@ def fired_upon?
     @fired_upon
   end
 
-  def render(*)
-    if !@ship.nil? && @fired_upon == false
+  def render(player = false)
```

Comment:
The * wasn't working because with the * it was trying to run a multiplication operation so I replaced with player and it worked like a charm